### PR TITLE
fixup-mountpoints

### PR DESCRIPTION
**Pull request by dsankouski**
Change-Id: Ib05034ce4c4735cacd56805205f8a1089f430234

### CODE REVIEW

File: fixup-mountpoints [hunk 1]
```diff
@@ -183,6 +183,16 @@ case "$DEVICE" in
...
+    "a5y17lte")
+    sed -i \
+        -e 's platform/13540000.dwmmc0/by-name/SYSTEM mmcblk0p19 ' \
+        -e 's platform/13540000.dwmmc0/by-name/CACHE mmcblk0p20 ' \
+        -e 's platform/13540000.dwmmc0/by-name/USERDATA mmcblk0p25 ' \
+        -e 's platform/13540000.dwmmc0/by-name/EFS mmcblk0p3 ' \
+        -e 's platform/13540000.dwmmc0/by-name/HIDDEN mmcblk0p21 ' \
+        "$@"
+    ;;
+
```
Comment (jbruechert) [inline fixup-mountpoints]: Please remove the unused spaces in this line.

**Review by jbruechert — CHANGES_REQUESTED** [review]

File: fixup-mountpoints (hunk 1 above)
Comment (dsankouski) [reply fixup-mountpoints]: done

File: fixup-mountpoints [hunk 2]
```diff
@@ -183,8 +183,18 @@ case "$DEVICE" in
...
+            -e 's platform/13540000.dwmmc0/by-name/CACHE mmcblk0p20 ' \
+            -e 's platform/13540000.dwmmc0/by-name/USERDATA mmcblk0p25 ' \
+            -e 's platform/13540000.dwmmc0/by-name/EFS mmcblk0p3 ' \
+            -e 's platform/13540000.dwmmc0/by-name/HIDDEN mmcblk0p21 ' \
+            "$@"
+        ;;
+
     "moto_msm8960_jbbl")
-       sed -i \
+        sed -i \
```
Comment (vanyasem) [inline fixup-mountpoints]: I am not sure if changing some other device is necessary in this specific PR

**Comment by vanyasem** [comment]
why have you closed it?

**Comment by dsankouski** [comment]
Because I amended commit.

**Comment by dsankouski** [comment]
I've created new pull request

**Pull request closed without merging**